app/konnect/dev-portal/customization/self-hosted-portal.md

```diff
@@ -0,0 +1,37 @@
+---
+title: About Self-Hosted Dev Portal
+content_type: explanation
+badge: oss
+---
+
+In {{site.konnect_saas}}, you have two hosting options for your Dev Portal: a cloud hosted Dev Portal with {{site.konnect_short_name}} or a self-hosted, open source Dev Portal. With the cloud hosted Dev Portal in {{site.konnect_short_name}}, your Dev Portal is hosted for you and it simplifies your deployment experience. The self-hosted, open source Dev Portal provides all the same features as the cloud-hosted portal, it just also gives you control over your hosting service and where your Dev Portal is hosted. 
+
+You can use the open source Dev Portal to display your APIs to developers on a self-hosted website. This page explains the benefits of using a self-hosted Dev Portal, a high-level overview of how the self-hosted portal works, and how to enable it.
+
+## Self-hosted Dev Portal benefits
+
+There are several benefits to keep in mind when deciding whether to use a {{site.konnect_short_name}}-hosted or self-hosted Dev Portal. The self-hosted portal provides the following benefits: 
+
+* **Fully customizable:** Use the [example frontend Dev Portal application](https://github.com/Kong/konnect-portal) as a starting point and then customize Dev Portal for your needs using the Portal API. You can also integrate the API specs with workflows tailored to your organization's own processes.
+* **Hosting service choice:** When you self-host, you also get to choose which hosting service you use to deploy your Dev Portal. 
+* **Range of customization options:** With the self-hosted Dev Portal, you determine how much you want to customize. You can choose to use the [example application](https://github.com/Kong/konnect-portal) right out of the box or you can use the Portal API to have more fine-grained control.
```

Comment:
```suggestion
* **Range of customization options:** With the self-hosted Dev Portal, you determine how much you want to customize. You can choose to use the example application right out of the box or you can use the Portal API to have more fine-grained control.
```
Feels odd to link this a second time with a different name because as the reader I felt like I was going to be taken to a different link -- I already had the repo tab open from the first bullet's link. You can either remove the link, or keep the link but call it "example front end dev portal application" like you did in the first bullet.